backend/src/jobs/automated_reminders.ts

```diff
@@ -33,20 +33,58 @@ export const handler = async () => {
   app.setViewEngine('njk');
 
   const recommitmentService = app.get(RecommitmentService);
-  const recommitment_cycle = await recommitmentService.checkRecommitmentPeriod()
-  const today = new Date(datePST(new Date()))
-  const testEmails = process.env.TEST_EMAIL.split(',')
-  if (today < recommitment_cycle.endDate || today > recommitment_cycle.startDate) {
-    const data = process.env.ENV !== 'production'
-    ? await recommitmentService.handleSendAutomatedReminders(true, testEmails)
-    : await recommitmentService.handleSendAutomatedReminders();
-    logger.log('Sent out automated reminders')  
-    console.log(data)
+  const recommitment_cycle =
+    await recommitmentService.checkRecommitmentPeriod();
+
+  const today = datePST(new Date())
+
+  logger.log(`Automated Reminders Job: ${today}`, 'Recommitment');
+
+  if (
+    today < recommitment_cycle.endDate ||
+    today > recommitment_cycle.startDate
+  ) {
+    if (process.env.ENV !== 'production') {
+      const testEmails = process.env.TEST_EMAIL.split(',');
```

Comment:
Do we need an || clause here in case there's no TEST EMAIL env?

Reply:
Good call - mail service will error if we try to send to undefined so I'll return instead